### ISSUE TITLE
Would be beautiful to have `combine-output-ports` procedure.

### ISSUE THREAD

**Issue by EligiusSantori** [racket/racket#2724]
With specs:
```
(combine-output-ports what . with) → output-port?
    what : output-port?
    with : list of output-port?
```

As example it very useful in this case:
```
(with-logging-to-port ((combine-output-ports
                        (current-error-port)
                        (open-output-file "my.log"))
    MY-SYNTAX
)
```
Procedure `make-output-port` is too unfriendly for this purposes.
I wonder why such procedure isn't already exists in `racket/port` package.

**Comment by lexi-lambda** [comment]
I agree that such a function would be useful to have in `racket/port`, but in the meantime, it might help you to look at `make-pipe` and `copy-port`, which together make an implementation of this function pretty easy since `copy-port` accepts multiple output ports.

**Comment by lkh01** [comment]
Unless @EligiusSantori wants to, I could implement this as my first contribution to racket :)
Seems straightforward.

**Comment by sorawee** [comment]
Closed by #2726
